api/src/types.rs

```diff
@@ -429,3 +542,18 @@ pub struct PoolInfo {
 	/// Total size of pool + orphans
 	pub total_size: usize,
 }
+
+#[test]
+fn serialize_printable_commitment() {
+	let hex_output = "{\
+        \"output_type\":\"Coinbase\",\
+        \"commit\":\"083eafae5d61a85ab07b12e1a51b3918d8e6de11fc6cde641d54af53608aa77b9f\",\
+        \"switch_commit_hash\":\"85daaf11011dc11e52af84ebe78e2f2d19cbdc76000000000000000000000000\",\
```

Comment:
@antiochp I observed that the hex values that were being outputted were only 160 bits long while the switch commit hash is supposed to be 256 bits long. In this implementation, the switch commit length is 256 bits as per `transaction.rs`.